src/test_server.py

```diff
@@ -1,79 +1,117 @@
 # -*- coding: utf-8 -*-
 """Test file performs tests on client.py and server.py."""
 from __future__ import unicode_literals
-from server import response_ok, response_error, parse_message
 import pytest
+try:
+    from http.client import HTTPException
+except ImportError:
+    from httplib import HTTPException
 
 CRLF = '\r\n'
 
 
-def test_parse_message_good(valid_200_response):
-    """Testing parse_message with a valid_200_response"""
-    header_lines, body = parse_message(valid_200_response)
-    print('header lines: {}'.format(header_lines))
-    print('body: {}'.format(body))
-    assert 1 == 1
+# --------------------------------------------------------------------
+# Start parse_message tests
+# --------------------------------------------------------------------
+
+
+def test_parse_request_no_clrfclrf(invalid_request):
```

Comment:
If you aren't going to use the data returned by the `invalid_request` fixture, you certainly don't need to use it in this test.